tests/isar/services/utilities/test_scheduling_utilities.py

```diff
@@ -1,16 +1,52 @@
+from contextlib import nullcontext as does_not_raise
+
 import pytest
+from fastapi import HTTPException
+from pytest_mock import MockerFixture
 
 from isar.models.communication.queues import QueueIO, QueueTimeoutError
 from isar.services.utilities.queue_utilities import QueueUtilities
+from isar.services.utilities.scheduling_utilities import SchedulingUtilities
+from isar.state_machine.states_enum import States
+from tests.mocks.mission_definition import MockMissionDefinition
 
 
-def test_timeout_send_command(mocker, scheduling_utilities):
+def test_timeout_send_command(
+    mocker: MockerFixture, scheduling_utilities: SchedulingUtilities
+):
     mocker.patch.object(QueueUtilities, "check_queue", side_effect=QueueTimeoutError)
     q: QueueIO = QueueIO(input_size=1, output_size=1)
     with pytest.raises(QueueTimeoutError):
         scheduling_utilities._send_command(True, q)
     assert q.input.empty()
 
 
-import queue
-from queue import Queue
+@pytest.mark.parametrize(
+    "capabilities, expectation",
+    [
+        (["drive_to_pose"], pytest.raises(HTTPException)),
+        (["drive_to_pose", "take_image"], does_not_raise()),
+    ],
+)
+def test_assert_robot_is_capable_of_mission(
```

Comment:
Consider splitting into two test to avoid complexity from does_not_raise()

Reply:
Good idea, implemented